components/eamxx/cime_config/testdefs/testmods_dirs/scream/output/diags/shell_commands

```diff
@@ -29,7 +29,7 @@ $YAML_EDIT_SCRIPT -g \
  --avg-type INSTANT               \
  --freq HIST_N                    \
  --freq-units HIST_OPTION         \
- --prefix ${CASE}.scream.diags.hi \
+ --prefix ${CASE}.scream.diags.h  \
```

Comment:
You may want to try it out, but I _think_ that CIME may expect the hist regex right after the model component name, so we may need `scream.h.diags`. Something to keep in mind if this does not work.

Reply:
I tried it out, the new regex works.